internal/profile/_static/config.yml.example

```diff
@@ -10,6 +10,10 @@
 # Region where the Serverless project is going to be created
 # stack.serverless.region: aws-us-east-1
 
+## Enable apm-server
+# Flag to enable apm-server in elastic-package stack profile config
+# stack.apm_server_enabled: true
```

Comment:
Could you add some test that leverages this option? It would be probably enough something like `scripts/test-stack-command.sh`, but with this setting enabled, and maybe checking that the APM server is listening.

The steps to add an integration test for this, would be:
- Add a script to `scripts`, or parameterize one of the available ones.
- Add a Makefile target that calls the script.
- Add a CI stage in `.buildkite/pipeline.trigger.integration.tests.sh`.

Reply:
added a test that ensures APM Server is listening, thanks for the clear instructions